### ISSUE TITLE
Huge delays when returning to main menu after watching a TV Show

### ISSUE THREAD

**Issue by Kepners** [boxeehacks/boxeehack#491]
Huge delays when returning to main menu after watching a TV Show. This is only a BoxeePlus issue and not a BoxeeBox issue as my wife has a Boxeebox in the bedroom (unhacked) and complains to me that the Hacked Boxee takes ages to return to menus!

And I knows shes right i have noticed it.

**Comment by siewers** [comment]
I too have this problem, although I cannot verify whether it's an issue with BoxeeHack.
The time it takes for it to return to the main menu seems to depend on whether you watch the episode until it's complete (end credits etc.) or you stop playback in the middle of the show.
This could indicate an issue with the buffer needing to flush perhaps?

**Comment by Mike120** [comment]
I have this issue also. I deal with it by pausing the show just before it's about to end and just backing out. This seems to work most of the time.

**Comment by jampeef** [comment]
Yeah i have this sometimes also. if it plays to the end it returns just fine.
